.github/workflows/runTests.yml

```diff
@@ -0,0 +1,28 @@
+name: Run tests
```

Comment:
is this borrowed from a template / tutorial somewhere?

Reply:
Yup! If you start to create a new action you'll see the android template. I added gradle caching and changed the build command from `./gradlew build` to what we have here, running tests only. (I believe `build` would try to publish the libraries, which would fail every time without the secrets.)